packages/wonder-blocks-dropdown/src/components/__tests__/dropdown-core.test.js

```diff
@@ -317,7 +315,7 @@ describe("DropdownCore", () => {
         dropdown.simulate("keydown", {keyCode: keyCodes.down});
         dropdown.simulate("keyup", {keyCode: keyCodes.down});
         jest.runAllTimers();
-        expect(document.activeElement).toBe(elementAtIndex(dropdown, 1));
+        expect(document.activeElement).toBe(elementAtIndex(dropdown, 2));
```

Comment:
Do you know why the focus changed to a different element in this case?

Reply:
the initialFocusedIndex is 1 so keying down should focus 2 so this assertion is logical to me.

I would assume it's either because the click behavior ignored the "initialFocusedIndex" prop.

We could probably remove this assertion altogether since it's logic that doesn't really do much.